src/main/docker/atlassian.yml

```diff
@@ -2,7 +2,7 @@ version: "2"
 services:
     jira:
         container_name: artemis_jira
-        image: atlassian/jira-core:8.15.3
+        image: atlassian/jira-core:8.13.5
```

Comment:
Is this large downgrade really necessary?
At least for me, 8.15.0 works just fine and is only a slight downgrade.
You could of course also try to update to the latest version, which appears to be [8.17.1](https://hub.docker.com/layers/atlassian/jira-core/8.17.1/images/sha256-29831e4d61e055e14cad1a30b07a42b3646d21a7c0c4db8f9ec1b5bf1b4dea85?context=explore)

Reply:
for me, when using 15.3 I got the error message "manifest for atlassian/jira-core:8.15.3 not found: manifest unknown: manifest unknown", I also investigated and the seems that there is no image with tag 15.3. So I'd say it's not a downgrading but rather fixing a type. At least this is what @FlorianGlombik and me concluded when talking about my issue recently.

Reply:
I am aware that 8.15.3 does not work. That's why I said "8.15.**_0_**" works fine for me.
I understand the need to change this version, but I do not think we need to downgrade by two minor versions when a downgrade by three patches also works.

Reply:
The reason for `8.15.3` seems to be a simple typo when I updated the version the last time it should have been `8.13.5`.
I propose to use `8.13.13` as this is the version that we are currently using in production for [jira](https://jira.ase.in.tum.de/), and I think that it is desirable to keep the local setup as close as possible to the production environment.
_In other words, I would not consider this as a downgrade as it was previously an unintended upgrade to another version._

Reply:
I also updated the Bitbucket version accordingly to `7.17.1`